service/tracker/execution.go

```diff
@@ -171,6 +189,45 @@ func (e *Execution) processNext() error {
 	return nil
 }
 
+func (e *Execution) checkTrieUpdates(record *uploader.BlockData, blockID flow.Identifier) error {
```

Comment:
I think you can use the `TrieUpdates` as the argument since we only use that in the function.